dev/NumberBox/NumberBox.cpp

```diff
@@ -71,11 +72,8 @@ void NumberBox::OnApplyTemplate()
         return textBox;
     }());
 
-    // Initializing precision formatter. This formatter works neutrally to protect against floating point imprecision resulting from stepping/calc
-    m_stepPrecisionFormatter.FractionDigits(0);
-    m_stepPrecisionFormatter.IntegerDigits(1);
-    m_stepPrecisionFormatter.NumberRounder(nullptr);
-    m_stepPrecisionRounder.RoundingAlgorithm(winrt::RoundingAlgorithm::RoundHalfAwayFromZero);
+    // .NET rounds to 12 significant digits when displaying doubles, so we will do the same.
+    m_displayRounder.SignificantDigits(12);
```

Comment:
>SignificantDigits [](start = 21, length = 17)

Does this mean we will always display 12 sig figs, even for something like (1 / 2) = 0.500000000000?

Reply:
No, that would be awful. :) It just never shows more than 12.

Reply:
how many does it show for 1/3?

---
In reply to: [345435018](https://github.com/microsoft/microsoft-ui-xaml/pull/1602#discussion_r345435018) [](ancestors = 345435018)

Reply:
You get 12 3s. "0.333333333333"